CONTRIBUTING.md

```diff
@@ -132,13 +132,12 @@ CSS
 ```
 
 Here is a full table of available CSS variables:
-
 | Variable | Description | Use |
-|-|-|-|
-| `--darkreader-neutral-background` | Neutral background color that corresponds to the user's settings. | Mostly used for elements that have a wrong background color |
-| `--darkreader-neutral-text` | Neutral text color that corresponds to the user's settings. | Used for elements with a wrong text color |
-| `--darkreader-selection-background` | The text color setting defined by the user. | The user's Text Color setting |
-| `--darkreader-selection-text` | The background color setting defined by the user. | The user's Background Color setting |
+|---|---|---|
+| **`--darkreader-neutral-background`** | Neutral background color that <br>corresponds to the user's settings. | Mostly used for elements that have <br>a wrong background color |
```

Comment:
Same as above.

Reply:
Consistency with above, although was able to also use the backtick (code) style with these.

Reply:
👍🏽